client/lcd/swagger-ui/swagger.yaml

```diff
@@ -2365,8 +2365,8 @@ definitions:
         type: string
       shares:
         type: string
-      height:
```

Comment:
why was this deleted ?

Reply:
Looking at the LCD response, it didn't appear to be present inside the response itself. Am I wrong or was it an error?

Reply:
The entire swagger config needs to be refactored (responses are now: `{"height": X, "result": Y}`)

Reply:
So, can we resolve this and open up another issue?

Reply:
👍 mind opening it?